Sources/AlgoliaSearchClient/Helpers/Crypto/String+HMAC.swift

```diff
@@ -6,13 +6,24 @@
 //
 
 import Foundation
+
+#if os(Linux)
+import Crypto
+#else
 import CommonCrypto
+#endif
 
 extension String {
-
-  func hmac(algorithm: HmacAlgorithm, key: String) -> String {
+  
+  func hmac256(withKey key: String) -> String {
+    #if os(Linux)
+    let key = SymmetricKey(data: key.data(using: .utf8)!)
```

Comment:
Any way we can avoid for unwrapping in this function? I see 2

Reply:
It is safe to force-unwrap _unicode_ string-to-data transformation results.

But this should work as well.

```
    let key = SymmetricKey(data: Data(key.utf8))
    let digest = Array(HMAC<SHA256>.authenticationCode(for: Data(key.utf8), using: key))
```

Reply:
Fixed!